lib/hawkular/metrics/metric_api.rb

```diff
@@ -69,6 +69,18 @@ def query_stats(gauge_ids: [], counter_ids: [], avail_ids: [], rates: false, sta
       http_post(path, data)
     end
 
+    # Fetch all tags for metrics definitions
+    # @return [Hash{String=>String}]
+    def tags
+      tags = []
+      http_get('/metrics/').map do |g|
+        g['tags'].map do |k, v|
```

Comment:
We don't currently support single tag with multiple values. It's currently a solution dependant on how to do this, for example in Openshift integration we do with comma separated list. If there's a need for such in this solution, could you create a JIRA ticket about it? We could then look at implementing it in some sane way..